builder/bundles.go

```diff
@@ -364,13 +364,39 @@ func resolvePackages(numWorkers int, set bundleSet, packagerCmd []string, emptyD
 }
 
 func installFilesystem(packagerCmd []string, chrootDir string) error {
-	installArgs := merge(packagerCmd,
-		"--installroot="+chrootDir,
-		"install",
-		"filesystem",
-	)
-	_, err := helpers.RunCommandOutputEnv(installArgs[0], installArgs[1:], []string{"LC_ALL=en_US.UTF-8"})
-	return err
+	_, err := downloadRpms(packagerCmd, []string{"filesystem"}, chrootDir, 1)
+	if err != nil {
+		return err
+	}
+	path := chrootDir + "/var/cache/yum/clear/"
+	newPath, err := filepath.Glob(filepath.Join(path, "clear-*"))
+	if err != nil {
+		fmt.Print("cant find path to rpm")
+		return err
+	}
+	if len(newPath) > 0 {
+		newPath[0] = newPath[0] + "/packages"
+	} else {
+		return fmt.Errorf("cant find rpms packages")
+	}
+	rpmFull := "filesystem" + "*.rpm"
+
+	fullPath, err := filepath.Glob(filepath.Join(newPath[0], rpmFull))
+	if err != nil {
+		fmt.Println("cant find rpm path")
+		return err
+	}
+	for _, v := range fullPath {
+		if rpmMap[v] == 1 {
```

Comment:
Is this because of the source rpm? I am struggling to understand what is being skipped. It seems very sensitive to the rpm name being consistently ordered.

Reply:
pretty much she is adding the rpm name to a map and checking if it was already extracted to the chroot. This skip extraction if an rpm is part of multiple bundles (although in this particular case it should always be the first time filesystem is extracted so this check will be ignored).